main/auth/twitter.py

```diff
@@ -48,7 +47,9 @@ def signin_twitter():
 def retrieve_user_from_twitter(response):
   auth_id = 'twitter_%s' % response['id_str']
   user_db = model.User.get_by('auth_ids', auth_id)
-  return user_db or auth.create_user_db(
+  if user_db:
+    return user_db
+  return auth.create_user_db(
```

Comment:
isn't that essentially the same thing?

Reply:
It is, and my objective is to make the Auth methods similar, so it is easier to see what is done for each; see my comment at https://github.com/gae-init/gae-init/pull/1388 for my considerations.